app/controllers/AssessmentController.scala

```diff
@@ -64,16 +65,30 @@ class AssessmentController @Inject() (
                                             case Some(value) => form.fill(value)
                                             case None        => form
                                           }
-      } yield
+      } yield {
+        val areWeRecategorising = request.userAnswers.get(RecategorisingQuery(recordId)).getOrElse(false)
+
```

Comment:
I think this can be updated to

`val hasAssessmentBeenAnswered = request.userAnswers.get(AssessmentPage(recordId, index)).exists(_ == NotAnsweredYet)`

Reply:
oooh that's clever. Let's give it a try 🙂

Reply:
Oh was hoping it would kill the recategorising query too, but don't think it can 😢